evap/evaluation/migrations/0079_rename_staff_to_manager.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 2.0.7 on 2018-07-30 19:25
+
+from django.db import migrations, models
+
+
+def rename_group(apps, schema_editor, old_name, new_name):
+    Group = apps.get_model('auth', 'group')
+    db_alias = schema_editor.connection.alias
+    Group.objects.using(db_alias).filter(name=old_name).update(name=new_name)
```

Comment:
why is this `using` part necessary?

Reply:
This is the way it is suggested in the official documentation
https://docs.djangoproject.com/en/2.0/ref/migration-operations/#runpython